frontend/pages/artists.js

```diff
@@ -1,19 +1,41 @@
-import Layout from '../components/Layout';
 import fetch from 'isomorphic-unfetch';
 import React, {Component} from 'react';
 import {Config} from '../config';
-import ArtistIndex from '../components/ArtistIndex';
 import withLayout from '../decorators/withLayout';
+import Artist from '../components/Artist';
+
+const endpoint = `${Config.apiUrl}/wp-json/wp/v2/artists?_embed`;
+
+class ArtistIndex extends Component {
+  static async getInitialProps() {
+    const res = await fetch(endpoint);
+    const data = await res.json();
+    return {data};
+  }
 
-class ArtistIndexPage extends Component {
   render() {
+    const artists = this.props.data;
+    console.log(
```

Comment:
This works, I can log out the exact image url I need, but then when I try to put that into the {image} prop below it says that ['wp:featuredmedia'] is undefined, wtf